Reference/google/protobuf/unittest_mset.pb.swift

```diff
@@ -304,7 +304,7 @@ struct ProtobufUnittest_RawMessageSet: SwiftProtobuf.Message {
 
 extension Proto2WireformatUnittest_TestMessageSet {
   var ProtobufUnittest_TestMessageSetExtension1_messageSetExtension: ProtobufUnittest_TestMessageSetExtension1 {
-    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension)}
+    get {return getExtensionValue(ext: ProtobufUnittest_TestMessageSetExtension1.Extensions.message_set_extension) ?? ProtobufUnittest_TestMessageSetExtension1()}
```

Comment:
Double-checking that I understand where you're headed:

I think you're saying that you want to soon change this to be
```
      ... ?? ProtobufUnittest_TestMessageSetExtension1.defaultInstance
```
So that each generated message publishes a single `defaultInstance` to use here and in other similar situations?

We could make that change post-1.0 _except_ for the message types in the library itself.